HomeWork/Program.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System.Numerics;
+using System.Security.Cryptography;
+
+Console.WriteLine("Enter the number of array elements");
+string size = Console.ReadLine();
+
+Console.WriteLine();
+
+int[] elements = new int[Convert.ToInt32(size)];
```

Comment:
If size is not a number we will get an exception, it's better to use Int32.TryParse()